package.json

```diff
@@ -28,8 +28,8 @@
   "license": "MIT",
   "dependencies": {
     "async": "2.5.0",
-    "aws-sdk": "^2.408.0",
-    "lodash": "^4.17.15",
+    "aws-sdk": "^2.441.0",
+    "lodash": "4.17.4",
```

Comment:
Why downgrade to lodash?

Reply:
Good catch. I think this was a rebase issue. I just pushed a fix.